cpp/src/arrow/filesystem/gcsfs.h

```diff
@@ -40,24 +40,43 @@ struct ARROW_EXPORT GcsOptions {
   bool Equals(const GcsOptions& other) const;
 };
 
+/// - TODO(ARROW-1231) - review this documentation before closing the bug.
```

Comment:
```suggestion
// - TODO(ARROW-1231) - review this documentation before closing the bug.
```

(avoid emitting this in the rendered docstring)

Reply:
Done. Thanks.